plugins/modules/aci_interface_policy_spanning_tree.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = r'''
+---
+module: aci_interface_policy_spanning_tree
+short_description: Manage spanning tree interface policies (stp:IfPol)
+description:
+- Manage spanning tree interface policies on Cisco ACI fabrics.
+options:
+  stp_policy:
+    description:
+    - Name of the STP policy.
```

Comment:
Change into: The name of the STP policy.

Reply:
Done.